gcpy/benchmark.py

```diff
@@ -713,20 +713,31 @@ def make_benchmark_conc_plots(
     refds = reader(ref, drop_variables=gcon.skip_these_vars).load()
     devds = reader(dev, drop_variables=gcon.skip_these_vars).load()
 
+    # Rename SpeciesConc_ to SpeciesConcVV_ for consistency with new
+    # naming introduced in GEOS-Chem 14.1.0
+    for v in refds.data_vars.keys():
+        if v.startswith('SpeciesConc_'):
+            spc = v.replace('SpeciesConc_', '')
+            refds = refds.rename({v: 'SpeciesConcVV_' + spc})
+    for v in devds.data_vars.keys():
+        if v.startswith('SpeciesConc_'):
+            spc = v.replace('Species_Conc', '')
```

Comment:
Typo here: `Species_Conc` should be `SpeciesConc_``

Reply:
Thanks for catching that. The typo is now fixed in [ae12821f](https://github.com/geoschem/gcpy/commit/ae12821fdb30aaf47d90d38c28079159a822f80b).